### PR TITLE
progressIndicator: Change to built in ActivityIndicator component

### PR DESCRIPTION
**Pull request by carizzuti**
I changed the loading indicator to use the built in `ActivityIndicator` rather than the external library version.

image

### CODE REVIEW

File: src/components/container/Container.js [hunk 1]
```diff
@@ -21,11 +21,11 @@ export const Container = ({
     return (
       <SafeAreaView
       style={[
-        containerStyle, style.circleProgressIndicator,
+        containerStyle, style.circleActivityIndicator,
         {backgroundColor: background, paddingHorizontal: horizontalPadding},
       ]}
       {...rest}>
-        <ProgressIndicator.Circle size={size.xl} indeterminate={true} />
+        <ActivityIndicator size='large' color={color.primary} />
```
Comment (Dulaya) [inline src/components/container/Container.js:28]: Should it be size={size.lg} instead?

**Review by Dulaya — APPROVED** [review]
It was working fine on my end with the external library. So if you're using the native ActivityIndicator, it should works too.

File: src/components/container/Container.js (hunk 1 above)
Comment (carizzuti) [reply src/components/container/Container.js:28]: It's not using the size file. The ActivityIndicator has a built in size prop that takes 'small' or 'large'. It is very easy to switch back to the external library version that gives some more size flexibility if required.

**Review by Nap1400 — APPROVED** [review]
Looks fine to me.

**Review by lasharela — APPROVED** [review]

**Pull request merged**